haskell/gtk-gui/package.yaml

```diff
@@ -0,0 +1,59 @@
+name:                gtk-gui
+version:             0.1.0.0
+github:              "githubuser/gtk-gui"
+license:             BSD-3-Clause
+author:              "Author name here"
+maintainer:          "example@example.com"
```

Comment:
We should update this info, the author is Mark Karpov but we can change the maintainer.